Test/XLSXWriterTest.php

```diff
@@ -152,7 +152,6 @@ public function testFreezeCells($freeze_cols, $freeze_rows, $expected_active_cel
 
 		$selections = $sheet_view->selection;
 		for ($i = 0; $i < count($expected_active_cells); $i++) {
-			$this->assertNotEmpty($selections[$i]);
```

Comment:
We cannot assert that an object is not empty anymore.  But, since we are validating the internal values of this object, I think that we are safe just removing this assert
<img width="915" alt="image" src="https://user-images.githubusercontent.com/12685991/189985934-1d773ee1-8a8e-4f21-b32e-28f257595e2a.png">